relnote(Fx145): -webkit-fill-available value supported (#41639)
mdn/content: files/en-us/mozilla/firefox/releases/145/index.md

<!-- #### Removals -->

### CSS

> - Firefox now supports the legacy [`-webkit-fill-available` keyword](Web/CSS/WebKit_Extensions) as a value for the CSS {{cssxref("width")}} and {{cssxref("height")}} properties, to improve rendering of content on the web that uses this keyword.

Reviewer: I believe the main idea behind supporting other vendor's prefixed value is to improve web compatibility.

```suggestion
- Firefox now supports the legacy [`-webkit-fill-available` keyword](Web/CSS/WebKit_Extensions) as a value for the CSS {{cssxref("width")}} and {{cssxref("height")}} properties, to improve web compatibility.
```
